### PR TITLE
Update ft/eval/inf spec

### PR DESCRIPTION
**Pull request by rjaincc**
Update ft/eval/inf spec for vision, multimodal and automl

### CODE REVIEW

**Comment by novaturient95** [comment]
asset valdn run: https://github.com/Azure/azureml-assets/actions/runs/7538101517/job/20518081809?pr=2079

**Comment by novaturient95** [comment]
new valdn run: https://github.com/Azure/azureml-assets/actions/runs/7568279183/job/20609162806?pr=2079

**Comment by novaturient95** [comment]
finiteautomato spec updated with PR: #2144

File: assets/models/system/automl-ner/spec.yaml [hunk 1]
```diff
@@ -35,61 +34,58 @@ tags:
...
+      Standard_NC64as_T4_v3,
+      Standard_NC24ads_A100_v4,
+      Standard_NC48ads_A100_v4,
+      Standard_NC96ads_A100_v4,
+      Standard_ND96asr_v4,
+      Standard_ND96amsr_A100_v4,
+      Standard_ND40rs_v2
     ]
-version: 1
+version: 2
```
Comment (novaturient95) [inline assets/models/system/automl-ner/spec.yaml:91]: we can either increment version of all models with property changes in this PR or do it separately to make sure we dont increment model version more than once

File: assets/models/system/compvis-stable-diffusion-v1-4/spec.yaml [hunk 2]
```diff
@@ -23,11 +22,8 @@ tags:
...
-      Standard_NC24ads_A100_v4,
-      Standard_NC48ads_A100_v4,
-      Standard_NC96ads_A100_v4,
       Standard_ND40rs_v2,
       Standard_ND96amsr_A100_v4,
       Standard_ND96asr_v4
     ]
-version: 6
\ No newline at end of file
+version: 7
```
Comment (novaturient95) [inline assets/models/system/compvis-stable-diffusion-v1-4/spec.yaml:29]: nit: new line at EOF

File: assets/models/system/mmeft/spec.yaml [hunk 3]
```diff
@@ -26,62 +36,64 @@ tags:
...
+      Standard_NC4as_T4_v3, 
+      Standard_NC6s_v3, 
+      Standard_NC8as_T4_v3, 
       Standard_NC12s_v3,
-      Standard_NC16as_T4_v3,
-      Standard_NC24s_v3,
-      Standard_NC64as_T4_v3,
+      Standard_NC16as_T4_v3, 
+      Standard_NC24s_v3, 
+      Standard_NC64as_T4_v3,
```
Comment (novaturient95) [inline assets/models/system/mmeft/spec.yaml:91]: nit: can format yaml to remove whitespaces

File: assets/models/system/ocsort_yolox_x_crowdhuman_mot17-private-half/spec.yaml [hunk 4]
```diff
@@ -46,13 +41,11 @@ tags:
...
       Standard_NC64as_T4_v3,
       Standard_NC8as_T4_v3,
       Standard_NC96ads_A100_v4,
       Standard_ND40rs_v2,
       Standard_ND96amsr_A100_v4,
       Standard_ND96asr_v4
     ]
-version: 4
\ No newline at end of file
+version: 5
```
Comment (novaturient95) [inline assets/models/system/ocsort_yolox_x_crowdhuman_mot17-private-half/spec.yaml:51]: nit: new line at EOF

File: assets/models/system/runwayml-stable-diffusion-v1-5/spec.yaml [hunk 5]
```diff
@@ -23,11 +22,8 @@ tags:
...
-      Standard_NC24ads_A100_v4,
-      Standard_NC48ads_A100_v4,
-      Standard_NC96ads_A100_v4,
       Standard_ND40rs_v2,
       Standard_ND96amsr_A100_v4,
       Standard_ND96asr_v4
     ]
-version: 6
\ No newline at end of file
+version: 7
```
Comment (novaturient95) [inline assets/models/system/runwayml-stable-diffusion-v1-5/spec.yaml:29]: nit: new line at EOF

File: assets/models/system/runwayml_stable_diffusion_inpainting/spec.yaml [hunk 6]
```diff
@@ -21,11 +20,8 @@ tags:
...
-      Standard_NC24ads_A100_v4,
-      Standard_NC48ads_A100_v4,
-      Standard_NC96ads_A100_v4,
       Standard_ND40rs_v2,
       Standard_ND96amsr_A100_v4,
       Standard_ND96asr_v4
     ]
-version: 5
\ No newline at end of file
+version: 6
```
Comment (novaturient95) [inline assets/models/system/runwayml_stable_diffusion_inpainting/spec.yaml:27]: nit: new line at EOF

File: assets/models/system/stabilityai-stable-diffusion-2-1/spec.yaml [hunk 7]
```diff
@@ -23,11 +22,8 @@ tags:
...
-      Standard_NC24ads_A100_v4,
-      Standard_NC48ads_A100_v4,
-      Standard_NC96ads_A100_v4,
       Standard_ND40rs_v2,
       Standard_ND96amsr_A100_v4,
       Standard_ND96asr_v4
     ]
-version: 6
\ No newline at end of file
+version: 7
```
Comment (novaturient95) [inline assets/models/system/stabilityai-stable-diffusion-2-1/spec.yaml:29]: nit: new line at EOF

File: assets/models/system/stabilityai-stable-diffusion-xl-refiner-1-0/spec.yaml [hunk 8]
```diff
@@ -21,11 +20,8 @@ tags:
...
-      Standard_NC24ads_A100_v4,
-      Standard_NC48ads_A100_v4,
-      Standard_NC96ads_A100_v4,
       Standard_ND40rs_v2,
       Standard_ND96amsr_A100_v4,
       Standard_ND96asr_v4
     ]
-version: 2
\ No newline at end of file
+version: 3
```
Comment (novaturient95) [inline assets/models/system/stabilityai-stable-diffusion-xl-refiner-1-0/spec.yaml:27]: nit: new line at EOF

**Review by novaturient95 — APPROVED** [review]

**Review by MadhuM02 — APPROVED** [review]

File: assets/models/system/automl-ner/spec.yaml (hunk 1 above)
Comment (rjaincc) [reply assets/models/system/automl-ner/spec.yaml:91]: Let's do it in this PR itself. And, not o it later.

**Review by novaturient95 — APPROVED** [review]

**Review by gauravrajguru — APPROVED** [review]

**Review by novaturient95 — APPROVED** [review]

**Review by nvijayrania — APPROVED** [review]

**Pull request merged**